docs/users_guide/ocean/test_groups/baroclinic_gyre.rst

```diff
@@ -0,0 +1,180 @@
+
+Consider the Sverdrup transport:
+
+.. math:: \rho v_{\rm bt} = \hat{\boldsymbol{k}} \cdot \frac{ \nabla  \times \vec{\boldsymbol{\tau}}}{\beta}
+
+If we plug in a typical mid-latitude value for :math:`\beta` (:math:`2 \times 10^{-11}` m\ :sup:`-1` s\ :sup:`-1`)
+and note that :math:`\tau` varies by :math:`0.1` Nm\ :sup:`2` over :math:`15^{\circ}` latitude,
+and multiply by the width of our ocean sector, we obtain an estimate of approximately 20 Sv.
+
+This scaling is obtained via thermal wind and the linearized barotropic vorticity equation),
+the depth of the thermocline :math:`h` should scale as:
+
+.. math:: h = \left( \frac{w_{\rm Ek} f^2 L_x}{\beta \Delta b} \right) ^2 = \left( \frac{(\tau / L_y) f L_x}{\beta \rho'} \right) ^2
+
+where :math:`w_{\rm Ek}` is a representive value for Ekman pumping, :math:`\Delta b = g \rho' / \rho_0`
+is the variation in buoyancy across the gyre,
+and :math:`L_x` and :math:`L_y` are length scales in the
+:math:`x` and :math:`y` directions, respectively.
+Plugging in applicable values at :math:`30^{\circ}`\ N,
+we obtain an estimate for :math:`h` of 200 m.
+
+.. _baroclinic_gyre.cfg:
+
+config options
+--------------
+
+All 2 test cases share the same set of config options:
+
+.. code-block:: cfg
+
+    # Options related to the vertical grid
+    [vertical_grid]
+    
+    # the type of vertical grid
+    grid_type = linear_dz
+    
+    # the linear rate of thickness increase for linear_dz
+    linear_dz_rate = 10.
+    
+    # Number of vertical levels
+    vert_levels = 15
+    
+    # Total water column depth
+    bottom_depth = 1800.
+    
+    # The type of vertical coordinate (e.g. z-level, z-star)
+    coord_type = z-star
+    
+    # Whether to use "partial" or "full", or "None" to not alter the topography
+    partial_cell_type = None
+    
+    # The minimum fraction of a layer for partial cells
+    min_pc_fraction = 0.1
+    
+    
+    # config options for MITgcm baroclinic gyre
+    [baroclinic_gyre]
+    # comment
+    lat_min = 15
+    lat_max = 75
+    lon_min = 0
+    lon_max = 60
+    
+    # Initial vertical temperature profile
+    initial_temp_top = 33.
+    initial_temp_bot = 1.
+    
+    # Constant salinity value (also used in restoring)
+    initial_salinity = 34.
+    
+    # Maximum zonal wind stress value
+    wind_stress_max = 0.1
+    
+    # Surface temperature restoring
+    restoring_temp_min = 0.
+    restoring_temp_max = 30.
+
+    # Restoring timescale for surface temperature (in days)
+    restoring_temp_timescale = 30.
+
+performance_test
+----------------
+
+``ocean/baroclinic_gyre/performance_test`` is the default version of the
+baroclinic_gyre test case for a short (3 time steps) test run and validation of
+prognostic variables for regression testing.
+
+3_year_test
```

Comment:
Somewhere here (or above), I would say that a user can run the `forward` step several times, advancing the simulation by 3 years each time.  Or that they can alter the namelist file to increase the duration of the simulation.

Reply:
Note however that you would need to make some changes I suggested to `forward.py` for this to actually work.